src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -357,16 +417,78 @@ size_t MKLDNNEmitter::build_quantized_convolution(const mkldnn::memory::desc& in
              mkldnn_utils::global_cpu_engine},
             *m_mkldnn_primitives[input_data_index],
             *m_mkldnn_primitives[weights_index],
-            *m_mkldnn_primitives[bias_index],
+            *m_mkldnn_primitives[quantize_deps[1]],
             *m_mkldnn_primitives[result_index]));
 
-        m_primitive_deps[conv_index] = {input_data_index, weights_index, bias_index, result_index};
+        m_primitive_deps[quantize_bias_reorder_index] = {
+            quantize_deps[0], quantize_deps[1], reordered_buf_index};
+        m_primitive_deps[conv_index] = {input_data_index, weights_index, result_index};
     }
     catch (const mkldnn::error& e)
     {
         throw ngraph_error("Could not create convolution " + e.message);
     }
-    return conv_index;
+    return std::vector<size_t>{conv_index, quantize_bias_reorder_index};
+}
+
+std::vector<size_t> MKLDNNEmitter::build_bias_reorder(const ngraph::Node* node,
+                                                      const std::vector<TensorViewWrapper>& args,
+                                                      const std::vector<TensorViewWrapper>& out,
+                                                      const mkldnn::memory::desc& reorder_bias_desc)
+{
+    auto convolution = static_cast<const ngraph::op::QuantizedConvolutionBias*>(node);
+
+    // For dilation, MKLDNN wants to know how many elements to insert between, not how far
+    // apart to space the elements like nGraph. So we have to subtract 1 from each pos.
+    ngraph::Strides window_dilation_strides_adjusted;
+
+    for (size_t s : convolution->get_window_dilation_strides())
+    {
+        window_dilation_strides_adjusted.push_back(s - 1);
+    }
+
+    auto data_desc = mkldnn_utils::get_input_mkldnn_md(node, 0);
+    auto weights_desc = mkldnn_utils::get_input_mkldnn_md(node, 1);
+
+    // MKLDNN relies on named formats for kernel selection
+    if (weights_desc.data.format == mkldnn_nchw)
+        weights_desc.data.format = mkldnn_oihw;
+    if (weights_desc.data.format == mkldnn_ncdhw)
+        weights_desc.data.format = mkldnn_oidhw;
+
+    auto result_desc = mkldnn_utils::get_output_mkldnn_md(node, 0);
+
+    mkldnn::post_ops ops;
+
+    auto add_relu = [&]() {
+        return (dynamic_cast<const ngraph::op::QuantizedConvolutionBias*>(node))->with_relu();
+    };
+
+    if (add_relu())
```

Comment:
Unless the lambda `add_relu` is used elsewhere, why define it.

instead this would work
```
if ((dynamic_cast<const ngraph::op::QuantizedConvolutionBias*>(node))->with_relu())
```

or:
```
bool add_relu = (dynamic_cast<const ngraph::op::QuantizedConvolutionBias*>(node))->with_relu();
if (add_relu)
```